packages/core/assistant/src/templates/assistant.tpl

```diff
@@ -15,13 +15,13 @@ Before responding, explain your reasoning and include your detailed chain-of-tho
 Include the following steps:
 
 - Analyze the structure and type of the content in the user's message.
-- Can you complete the task using the available artifact definitions?
-- If you can't complete the task using the available artifact definitions, query the list of available artifact definitions using the appropriate tool.
-- Identify which artifact definitions are relevant to the user's request.
+- Determine if you complete the task using the available blueprint definitions?
```

Comment:
Is this section still relevant? This calls out things like `require_artifact_definitions` tool. I belive we've moved away from that approach in favor of having the user bind blueprints manually.